cbits/cryptonite_sha3.c

```diff
@@ -126,7 +131,7 @@ void cryptonite_sha3_update(struct sha3_ctx *ctx, const uint8_t *data, uint32_t
 	}
 
 	if (need_alignment(data, 8)) {
-		uint64_t tramp[200 - 2 * (224 / 8)];
+		uint64_t tramp[200 - 2 * (128 / 8)];
```

Comment:
scratch that, I made the link with SHAKE 128 bits. I cleanup the buffer size constant and trampoline code to prevent future misunderstanding